components/webfield/SeniorAreaChairConsole.js

```diff
@@ -113,6 +117,21 @@ const SeniorAreaChairConsole = ({ appContext }) => {
         : Promise.resolve([])
       // #endregion
 
+      // #region getInvitations
+
+      const invitationsP = [reviewerAssignmentId, areaChairAssignmentId].map((invitation) =>
```

Comment:
can you make these API calls only if `reviewerAssignmentId` and `areaChairAssignmentId` are present?

otherwise current SAC console will fail because there is no parameter in the API call.

Reply:
I added a default value so no need to check this.